client/src/layout/navbar-layout/Navbar.module.scss

```diff
@@ -27,6 +27,14 @@
             cursor: pointer;
             margin-bottom: 0;
         }
+        .avatar{
```

Comment:
Please use plugins like prettier to format all the code before the pull request.